assets/scss/thema/lustrum.scss

```diff
@@ -12,70 +20,146 @@ $lustrum-kleur-wit: #eeeeee;
 	src: url('../../fonts/lustrum12/OrpheusPro-Bold.otf');
 }
 
-$headings-font-family: 'Orpheus Pro', 'Source Sans Pro', Helvetica, Arial, sans-serif;
+$theme-colors: (
+	"primary": $kleur-primair,
+	"danger": #E54B4B,
+	"white": $kleur-text,
+);
+
+/* kleuren: turned white and black upside down */
```

Comment:
Je kan ook het donker thema extenden. (als je een donker thema wil)

Reply:
Hahaha, is nu een beetje te laat. Heb ook niet heel veel uit het donker thema gepakt. Alleen de herdefinitie van de grijstinten. Echter heb ik die ook volgens mij nog een beetje aangepast.